src/main/java/nextstep/subway/domain/Sections.java

```diff
@@ -0,0 +1,118 @@
+package nextstep.subway.domain;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Objects;
+import java.util.OptionalInt;
+import java.util.stream.IntStream;
+
+import javax.persistence.Embeddable;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToMany;
+
+@Embeddable
+public class Sections {
+	@OneToMany
+	@JoinColumn(name = "section_id")
+	private List<Section> sections;
+	
+	public Sections() {
+		sections = new ArrayList<>();
+	}
+	
+	public List<Section> getSections() {
+		sections.sort(new Comparator<Section>() {
+			@Override
+			public int compare(Section s1, Section s2) {
+				if (s1.getSectionOrder() > s2.getSectionOrder()) {
+					return 1;
+				}
+				if (s1.getSectionOrder() < s2.getSectionOrder()) {
+					return -1;
+				}
+				return 0;
+			}
+		});
```

Comment:
요런식으로도 해볼 수 있을 것 같아요!
```suggestion
    sections.sort(Comparator.comparingInt(Section::getSectionOrder));
```

Reply:
Comparator 클래스를 사용하는 방법도 있었네요 ㅎㅎ 감사합니다